python/tvm/hybrid/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""Hybrid Programming APIs of TVM Python Package.
+
```

Comment:
need to add hybrid.rst to the python API, see example here 
https://github.com/dmlc/tvm/blob/master/docs/api/python/index.rst
https://github.com/dmlc/tvm/blob/master/docs/api/python/intrin.rst